config/webpack/webpack.common.js

```diff
@@ -117,7 +117,7 @@ const webpackConfig = ({envFile = '.env', platform = 'web'}) => ({
         rules: [
             // Transpiles and lints all the JS
             {
-                test: /\.js$/,
+                test: /\.(js|ts)x?$/,
                 loader: 'babel-loader',
```

Comment:
Do we not need to setup `ts-loader` here?

Reply:
@roryabraham TypeScript transpilation is handled by Babel.

There is `'@babel/preset-typescript'` added to the babel configuration file, and it is specified in the package.json